Resources/Prototypes/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml

```diff
@@ -596,3 +596,39 @@
   - type: Appearance
   - type: StaticPrice
     price: 750
+
```

Comment:
would be generally more readable if you added this entire block right after/near the disabler, instead the end of the file